.github/workflows/_linux-test.yml

```diff
@@ -212,8 +198,7 @@ jobs:
             -e SHARD_NUMBER \
             -e TEST_CONFIG \
             -e NUM_TEST_SHARDS \
-            -e PR_BODY \
```

Comment:
Nice!  Does this just fix this issue https://github.com/pytorch/pytorch/issues/103860 now that passing the PR_BODY is not needed anymore? :)